modules/site-health/audit-enqueued-assets/load.php

```diff
@@ -25,22 +25,33 @@ function perflab_aea_audit_enqueued_scripts() {
 		global $wp_scripts;
 		$enqueued_scripts = array();
 
-		foreach ( $wp_scripts->queue as $handle ) {
-			$src = $wp_scripts->registered[ $handle ]->src;
-			if ( $src && ! strpos( $src, 'wp-includes' ) ) {
+		foreach ( $wp_scripts->done as $handle ) {
+			$script = $wp_scripts->registered[ $handle ];
+
+			if ( $script->src && ! strpos( $script->src, 'wp-includes' ) ) {
```

Comment:
Personally adding an early return (`continue` in this case) would allow the removal for the indentation level here, giving a simpler way to follow the code.

```suggestion
			if ( ! $script->src || strpos( $script->src, 'wp-includes' ) === false ) {
					continue;
			}
```

Reply:
makes sense